plugins/lang/python/python_loader.cc

```diff
@@ -443,9 +443,13 @@ runPythonScript(std::shared_ptr<PythonContext> titl,
     // if we're simply executing the body of the script
     try {
         if(elementToRun == PackageElement::MAIN) {
-                JML_TRACE_EXCEPTIONS(false);
-                boost::python::object obj
-                    = boost::python::exec(scriptSourceStr, pyControl.main_namespace);
+            JML_TRACE_EXCEPTIONS(false);
+            int argc = 1;
```

Comment:
this should be sizeof(argv1) / sizeof(char *), and you can do const char * argv[] = { "arg1", "arg2" }.  That is clearer and less error prone.